### DISCUSSION TITLE
Difficulty adjust looks worse in the new Mode select.

### DISCUSSION THREAD

**Discussion by KoleckOLP** [ppy/osu#18621] [Ideas] [answered]
I don't know when this new mod select menu was released, and I think in most places it does look better than the old one.

But when I select "difficulty adjust" and I want to change the AR in the old one I saw all the options without them being cut off or needitng to scrool, 
in the new one it looks just plain worse.

There is a lot of empty space next to the difficulty adjust, I presume it's for when I have selected more things.

But I really don't like how cut off the text is.

Can anything be done so I can see the extend limits without scrooling?

OLD:
![old](https://user-images.githubusercontent.com/22729896/172607229-749346ed-ddcf-441a-ac32-2480cd30e476.png)

NEW:
![new](https://user-images.githubusercontent.com/22729896/172607244-9fb8bf9e-2fd8-4b07-8d00-49f3dbace8e2.png)

**Comment by bdach** [comment, marked as answer]
mostly duplicate of/overlaps with https://github.com/ppy/osu/discussions/18507
